### PR TITLE
add documentation for new post message in slack feature (thread id)

### PR DESCRIPTION
**Pull request by mathias-vandaele**
## Description

This is an update for the documentation, as we have added a new feature to the `post_message` function.
We can now add a specific thread, which correspond to the id of the message one wants to start a thread from

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory. (N/A)
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/slack.md [hunk 1]
```diff
@@ -89,7 +89,8 @@ To post a message, take the following steps:
 2. Set **Channel/User Name** to either the **channel** or **user** you want to send the message to.
    1. A **channel** is specified by a unique identifier starting with a `#` (for example, `#myChannel`).
    2. A **user** is specified by a username starting with an `@` symbol (for example, `@myUser`).
-3. Select a **Message type**.
+3. (Optional) A thread can be specified, to start a thread from a specific message, `ts` in the response can be used (see [here](#post-message))
```
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/slack.md:92]: thread => should be bold I guess? **thread**

Comment (mathias-vandaele) [reply docs/components/connectors/out-of-the-box-connectors/slack.md:92]: Yes it should, changing it right now

**Comment by akeller** [comment]
Is this part of the alpha3 release? If so, please add the label.

**Review by johnBgood — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Review by johnBgood — APPROVED** [review]

**Comment by christinaausley** [comment]
@chillleader @mathias-vandaele @sbuettner Fine with me merging this in for release tomorrow?

**Pull request merged**